Testscripts/Linux/SRIOV-VerifyVF-Connection.sh

```diff
@@ -25,6 +25,28 @@ fi
     exit 0
 }
 
+function version_lt() { test "$(echo "$@" | tr " " "\n" | sort -rV | head -n 1)" != "$1"; }
+
+MIN_VERSION_SUPPORTED="6.7"
```

Comment:
This is the fixed version in RHEL and CentOS? 
what about other distro? 
is it hit this line?

Reply:
Added a distro check

Reply:
if $DISTRO_NAME is sles or ubuntu, it jumps to false statement, where is none. it does not skip.

Reply:
In current implementation of LIS we are supporting SRIOV on centos, rhel, oracle 6.7 to 6.10 
SRIOV is not supported in 6.5 and 6.6 of centos, rhel, oracle.
Hence, this if blocked is added to check and skip if distro_version is < 6.7
for other distros there is no change in the test.